core/src/decl/Kokkos_Declare_HIP.hpp

```diff
@@ -46,7 +46,27 @@
 #define KOKKOS_DECLARE_HIP_HPP
 
 #if defined(KOKKOS_ENABLE_HIP)
-#include <Kokkos_HIP.hpp>
+#include <HIP/Kokkos_HIP.hpp>
+#include <HIP/Kokkos_HIP_Space.hpp>
+#include <HIP/Kokkos_HIP_DeepCopy.hpp>
+#include <HIP/Kokkos_HIP_Half_Impl_Type.hpp>
+#include <HIP/Kokkos_HIP_Half_Conversion.hpp>
+#include <HIP/Kokkos_HIP_Instance.hpp>
+#include <HIP/Kokkos_HIP_MDRangePolicy.hpp>
+#include <HIP/Kokkos_HIP_Parallel_Range.hpp>
+#include <HIP/Kokkos_HIP_Parallel_MDRange.hpp>
+#include <HIP/Kokkos_HIP_Parallel_Team.hpp>
+#include <HIP/Kokkos_HIP_SharedAllocationRecord.hpp>
+#include <HIP/Kokkos_HIP_UniqueToken.hpp>
+
+namespace Kokkos {
+namespace Experimental {
+using HIPSpace           = ::Kokkos::HIPSpace;
+using HIPHostPinnedSpace = ::Kokkos::HIPHostPinnedSpace;
+using HIPManagedSpace    = ::Kokkos::HIPManagedSpace;
+using HIP                = ::Kokkos::HIP;
+}  // namespace Experimental
+}  // namespace Kokkos
```

Comment:
Why aren't these in `<fwd/Kokkos_Fwd_HIP.hpp>`?

Reply:
What would I want to do that?

Reply:
Also, should we deprecate `Kokkos::Experimental::HIP*` to be removed in `Kokkos 5`?

Reply:
> Also, should we deprecate Kokkos::Experimental::HIP* to be removed in Kokkos 5?

That's still open to discussion.

Reply:
We could move `<fwd/Kokkos_Fwd_HIP.hpp>` but that means I would need to re-run the test suite with `Experimental::HIP` to ensure that `Experimental::HIP` still works.

Reply:
OK you don't have to change